### PR TITLE
docs: Add v2 tests scenario proposal

### PR DESCRIPTION
**Pull request by xakraz**

### CODE REVIEW

File: docs/misc/1-v2-testsScenario.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+2. Check Manual NextSuspendTime
+    * Unsuspended if needed
+        + set `desiredState=Running`
+    * Check before (up)
+        + deployment
+        + statefulSet
+        + CronJob-suspended
+        + CronJob-unsuspended
+    * Set `nextSuspendTime=DATE+10s`
```
Comment (eze-kiel) [inline docs/misc/1-v2-testsScenario.md:92]: the minimum granularity is 1min

File: docs/misc/1-v2-testsScenario.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+1. Check DailySuspendTime
+    * Unsuspended if needed
+        + set `desiredState=Running`
+    * Check before (up)
+        + deployment
+        + statefulSet
+        + CronJob-suspended
+        + CronJob-unsuspended
+    * Set `dalySuspendTime=DATE+10s`
```
Comment (eze-kiel) [inline docs/misc/1-v2-testsScenario.md:111]: same as for `Fearture: nextSuspendTime`

File: docs/misc/1-v2-testsScenario.md (hunk 1 above)
Comment (xakraz) [reply docs/misc/1-v2-testsScenario.md:92]: ok 👌🏻

File: docs/misc/1-v2-testsScenario.md (hunk 2 above)
Comment (eze-kiel) [reply docs/misc/1-v2-testsScenario.md:111]: here the format is [`time.Kitchen`](https://cs.opensource.google/go/go/+/refs/tags/go1.18:src/time/format.go;l=103)

File: docs/misc/1-v2-testsScenario.md (hunk 1 above)
Comment (eze-kiel) [reply docs/misc/1-v2-testsScenario.md:92]: here the format is [`time.RFC822Z`](https://cs.opensource.google/go/go/+/refs/tags/go1.18:src/time/format.go;l=97)

**Review by eze-kiel — APPROVED** [review]
LGTM

**Pull request merged**